### PR TITLE
Fixed a typo in the .gitignore which was preventing the gradle wrapper

### PR DESCRIPTION
**Pull request by DDevilISL**
from being included in the repository

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.3.1-all.zip
```
Comment (MiniDigger) [inline gradle/wrapper/gradle-wrapper.properties]: Please remove this change, we don't want to downgrade Gradle

File: .gitignore [hunk 2]
```diff
@@ -6,7 +6,6 @@ target/
 build/
 
 # IDEA
-*.iml
 .idea/
```
Comment (MiniDigger) [inline .gitignore]: Why did you make this change?

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (DDevilISL) [reply gradle/wrapper/gradle-wrapper.properties]: Gradle changed it when I ran the wrapper command and I didn't notice it

File: .gitignore (hunk 2 above)
Comment (DDevilISL) [reply .gitignore]: *.iml is covered by .idea/, so it's redundant, anyhow, I reversed it

**Pull request merged**